portfolio/src/main/webapp/script.js

```diff
@@ -17,7 +17,8 @@
  */
 function addRandomGreeting() {
   const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+      ['I spend a lot of time on YouTube.', 'I am watching 3 different anime shows right now.', 
```

Comment:
I think I've these changes before, were you able to merge the previous pull request?

Reply:
Just did it, I forgot!